code/datums/components/sign_language.dm

```diff
@@ -222,13 +237,16 @@
 	return SPELL_INVOCATION_ALWAYS_SUCCEED
 
 /// Signal proc for [COMSIG_LIVING_TREAT_MESSAGE]
-/// Stars out our message if we only have 1 hand free.
+/// Changes our message based on criteria.
```

Comment:
A bit nondescriptive of a comment, I think it'd be better if we'd keep it as at least saying when it changes our message- _What_ criteria?